compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -351,51 +363,58 @@ setConvolutionVectorDistributionConfig(IREE::GPU::TargetAttr target,
     workgroupTileSizes[oc] = 1;
   }
   for (int64_t ic : llvm::drop_end(convolutionDims->inputChannel)) {
-    workgroupTileSizes[ic] = 1;
+    reductionTileSizes[ic] = 1;
   }
   // Compute the M/N dimension tile size by multiply subgroup information.
   workgroupTileSizes[mDim] =
       schedule->mWarpCount * schedule->mTileCount * schedule->mSize;
   workgroupTileSizes[nDim] =
       schedule->nWarpCount * schedule->nTileCount * schedule->nSize;
 
-  // Follow the LLVMGPU convention of keeping all of the tile sizes in one list.
-  workgroupTileSizes[kDim] = schedule->kTileCount * schedule->kSize;
+  reductionTileSizes[kDim] = schedule->kTileCount * schedule->kSize;
 
   // Tile all filter loop dimensions to 1.
   for (int64_t filterDim : convolutionDims->filterLoop) {
-    workgroupTileSizes[filterDim] = 1;
+    reductionTileSizes[filterDim] = 1;
   }
 
-  TileSizesListType tileSizes;
-  tileSizes.push_back(workgroupTileSizes);
+  MLIRContext *context = op.getContext();
+  Builder b(context);
+  SmallVector<NamedAttribute, 1> attrs;
```

Comment:
You insert 2 elements just after this
```suggestion
  SmallVector<NamedAttribute, 2> attrs;
```